### DISCUSSION TITLE
about `now`, is it still maintained?

### DISCUSSION THREAD

**Discussion by wl1806** [vercel/vercel#4361] [General] [answered]
I found this page 
https://github.com/zeit/now/tree/master/examples, goes to `Deploying Examples`
and when I click download `now`
it goes to https://vercel.com/download 
the problem is there is only `npm i -g vercel` command
after i install vercel, i still cannot do the `now`  command

**Comment by beersky** [comment, marked as answer]
Hi,

Use `vercel` as the command instead of `now`.

**Reply by styfle** [reply]
This is correct! Sorry about the confusion, I updated the examples in PR #4383.

We recommend using the [Git Integration](https://vercel.com/docs/v2/git-integrations) so you can use `git push` to deploy instead of installing Vercel CLI.
